tests/unit/vec/vec.cpp

```diff
@@ -281,6 +281,54 @@ struct CompileTimeKernelCompare2D
     }
 };
 
+/** Compile-time test cases for divCeil and divExZero */
+struct CompileTimeKernelDivCeilAndDivExZero
+{
+    ALPAKA_FN_HOST_ACC void operator()() const
+    {
+        using namespace alpaka;
+
+        // Test divCeil with 1D vectors
+        constexpr auto vec1 = Vec{7};
+        constexpr auto vec2 = Vec{3};
+        static_assert(divCeil(vec1, vec2) == Vec{(7 + 3 - 1) / 3}); // (7 + 3 - 1) / 3 = 9 / 3 = 3
+
+        // Test divCeil with 3D vectors
+        constexpr auto vec3 = Vec{3, 7, 5};
+        constexpr auto vec4 = Vec{2, 3, 4};
+        static_assert(divCeil(vec3, vec4) == Vec{(3 + 2 - 1) / 2, (7 + 3 - 1) / 3, (5 + 4 - 1) / 4});
+        // (3 + 2 - 1) / 2 = 4 / 2 = 2
```

Comment:
please always comment above the line, the auto formation can make strange things